spec/ride_spec.rb

```diff
@@ -0,0 +1,78 @@
+require './lib/ride'
+
+class Ride
+  attr_reader :name,
+              :distance,
+              :loop,
+              :terrain
+
+  def initialize(name, distance, loop, terrain)
+    @name = name
+    @distance = distance
+    @loop = loop
+    @terrain = terrain 
+  end
+
+
+  it 'has a name' do
+    ride = Ride1.new('Walnut Creek Trail')
+    name = ride1.name
+    expect(name).to eq ('Walnut Creek Trail')
+  end  
+
+  it 'has a distance' do
+    distance = Distance.new(10.7)
+    distance = ride1.distance
+    expect(distance).to eq (10.7)
+  end  
+
+  it 'is a loop' do
+    loop = Loop.new(false)
+    distance = ride.distance
+    expect(loop).to eq (false)
+  end  
+
+  it 'has terrain' do
+    terrain = Terrain.new('hills')
```

Comment:
Same with these methods down here -- create the Ride object, and then test the attributes on that ride object